exporter/tanzuobservabilityexporter/config.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tanzuobservabilityexporter
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/config"
+)
+
+type TracesConfig struct {
+	// Wavefront Proxy URL of the form protocol://host[:port]
+	Endpoint string `mapstructure:"endpoint"`
+
+	// A default value to use if a span doesn't contain an "application" label
+	DefaultApplication string `mapstructure:"default_application"`
+
+	// A default value to use if a span doesn't contain a "service" or "service.name" label
```

Comment:
The span does not have a `service.name` it is in the `Resource`. I would not add this initially, users can use the `attributeprocessor` to do these kind of changes.

Reply:
Thanks for pointing us to the `attributeprocessor`. We'll remove `DefaultService` and `DefaultApplication` from our exporter config and point users to `attributeprocessor` if they want to override those values.

Reply:
So are you going to remove this?

Reply:
Yes - removed in aa2ffe.